mithril.d.ts

```diff
@@ -3,8 +3,8 @@
 /**
 * This is the module containing all the types/declarations/etc. for Mithril
 */
-declare module _mithril {
-	interface MithrilStatic {
+declare module Mithril {
+	interface Static {
```

Comment:
Question: would "Mithril" be a better name for this? An interface can share the same name as its parent namespace, and this is describing the Mithril `m` export itself as well as its static properties.